README.md

```diff
@@ -85,8 +85,8 @@ Benchmarking flat_map...
 Benchmarking map.flatten...
 
 Name                  ips        average    deviation         median
-map.flatten        989.80      1010.31μs    (±12.63%)       998.00μs
-flat_map           647.35      1544.75μs    (±10.54%)      1556.00μs
+map.flatten        989.80      1010.31ms    (±12.63%)       998.00ms
+flat_map           647.35      1544.75ms    (±10.54%)      1556.00ms
```

Comment:
I adjusted the `map.flatten` median value, since it would have been `1.00`, which seems...strange